src/components/admin/HearingToolbar.js

```diff
@@ -0,0 +1,99 @@
+import moment from 'moment';
+import React from 'react';
+import {injectIntl, FormattedMessage} from 'react-intl';
+
+import Button from 'react-bootstrap/lib/Button';
+import ButtonToolbar from 'react-bootstrap/lib/ButtonToolbar';
+import Col from 'react-bootstrap/lib/Col';
+
+import {canEdit, isOpen} from '../../utils/hearing';
+import {hearingShape, userShape} from '../../types';
+
+
+if (typeof window !== "undefined") {
+  // Only loading in browser context
+  require('assets/less/hel/hearing_form.less');  // eslint-disable-line
+}
+
+
+const DATE_FORMAT = "LLLL";
+
+
+class HearingToolbar extends React.Component {
+
+  constructor(props) {
+    super(props);
+    moment.locale("fi-FI");
+  }
+
+  render() {
+    const user = this.props.user;
+    const hearing = this.props.hearing;
+    if (!canEdit(user, hearing)) {
+      return null;
+    }
+
+    let statusLabel = "";
+    const openingTime = moment(hearing.open_at);
+    const actions = [
+      <Button bsStyle="default" onClick={this.props.onEdit} key="edit">
+        <FormattedMessage id="editHearing"/>
+      </Button>
+    ];
+    const open = isOpen(hearing);
+    if (open && hearing.published) {
+      statusLabel = <FormattedMessage id="published"/>;
+      actions.push(
+        <Button bsStyle="danger" onClick={this.props.onCloseHearing} key="close">
```

Comment:
Another option is to keep this as is, because unpublishing is a rare occurrence, and perhaps should be done via admins. Though closing a Hearing prematurely is a very rare need indeed, so I'm not sure if we should have a button for that either. Hmm.